src/backend/dx11/src/lib.rs

```diff
@@ -1082,8 +1082,12 @@ bitflags! {
         const RENDER_TARGETS = (1 << 1);
         const VERTEX_BUFFERS = (1 << 2);
         const GRAPHICS_PIPELINE = (1 << 3);
```

Comment:
what's the semantic of `GRAPHICS_PIPELINE` now that you have each of the stages in a separate flag?

Reply:
This says that _something_ in the pipeline changed. Be it any of the individual flags, the vertex shader, raster state, etc.